core/schema/function.graphqls

```diff
@@ -0,0 +1,126 @@
+scalar FunctionID
+
+extend type Query {
+  "Load a function by ID"
+  function(id: FunctionID!): Function!
+}
+
+type Function {
```

Comment:
See [my other comment](https://github.com/dagger/dagger/pull/5757/files#r1316272095) for why this doesn't have any `with*` fields